website/templates/log_list.mako

```diff
@@ -46,12 +46,17 @@
                                 <span class="contributor-anonymous">A user</span>
                             </span>
                                 <span data-bind="ifnot:log.anonymous">
+                                    <span data-bind="if: log.user.registered">
                                     <span data-bind="if: log.userURL">
                                         <a class="overflow" data-bind="text: log.userFullName || log.apiKey, attr: {href: log.userURL}"></a>
                                     </span>
                                     <span data-bind="ifnot: log.userURL">
                                         <span class="overflow" data-bind="text: log.userFullName"></span>
                                     </span>
+                                    </span>
+                                    <span data-bind="ifnot: log.user.registered">
+                                        Deactivated User
```

Comment:
Not certain that's correct; it appears to not work at all on the activity log.
<hr>
<img width="295" alt="screen shot 2016-05-23 at 13 14 33" src="https://cloud.githubusercontent.com/assets/5659262/15478469/5d3d3332-20e8-11e6-813f-0ee4f5cd4240.png">
==============
<img width="489" alt="screen shot 2016-05-23 at 13 14 43" src="https://cloud.githubusercontent.com/assets/5659262/15478468/5d3829fa-20e8-11e6-909d-bac9522725d7.png">